com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/ResolveClasspathsHandler.java

```diff
@@ -162,12 +166,43 @@ public void acceptSearchMatch(SearchMatch match) {
         }
         String[][] result = new String[2][];
         if (JavaRuntime.isModularProject(javaProject)) {
-            result[0] = JavaRuntime.computeDefaultRuntimeClassPath(javaProject);
+            result[0] = computeDefaultRuntimeClassPath(javaProject);
             result[1] = new String[0];
         } else {
             result[0] = new String[0];
-            result[1] = JavaRuntime.computeDefaultRuntimeClassPath(javaProject);
+            result[1] = computeDefaultRuntimeClassPath(javaProject);
         }
         return result;
     }
+
+    private static String[] computeDefaultRuntimeClassPath(IJavaProject jproject) throws CoreException {
+        IRuntimeClasspathEntry[] unresolved = JavaRuntime.computeUnresolvedRuntimeClasspath(jproject);
+        List<String> resolved = new ArrayList<>(unresolved.length);
+        for (int i = 0; i < unresolved.length; i++) {
+            IRuntimeClasspathEntry entry = unresolved[i];
+            if (entry.getClasspathProperty() == IRuntimeClasspathEntry.USER_CLASSES) {
+                IRuntimeClasspathEntry[] entries = JavaRuntime.resolveRuntimeClasspathEntry(entry, jproject);
+                for (int j = 0; j < entries.length; j++) {
+
```

Comment:
Please use the existing API: 
```java
IClasspathEntry.isTest() 
```

Reply:
Nicer to use isTest since photon is added to pom.xml